ml-agents-envs/mlagents_envs/tests/test_rpc_utils.py

```diff
@@ -176,16 +190,28 @@ def proto_from_steps_and_action(
 def test_process_pixels():
     in_array = np.random.rand(128, 64, 3)
     byte_arr = generate_compressed_data(in_array)
-    out_array = process_pixels(byte_arr, False)
+    out_array = process_pixels(byte_arr, 3)
     assert out_array.shape == (128, 64, 3)
     assert np.sum(in_array - out_array) / np.prod(in_array.shape) < 0.01
     assert (in_array - out_array < 0.01).all()
 
 
+def test_process_pixels_multi_png():
+    height = 128
+    width = 64
+    num_channels = 7
+    in_array = np.random.rand(height, width, num_channels)
+    byte_arr = generate_compressed_data(in_array)
+    out_array = process_pixels(byte_arr, num_channels)
+    assert out_array.shape == (height, width, num_channels)
+    assert np.sum(in_array - out_array) / np.prod(in_array.shape) < 0.01
+    assert (in_array - out_array < 0.01).all()
```

Comment:
Very minor, but numpy has a method for this:
```python
np.allclose(in_array, out_array)
```

Reply:
Copied from other tests, but I'll give that a try.

Reply:
Actually, all the other tests were kinda wonky because they weren't taking the absolute value of the difference :/  Update them all to use `np.allclose` with the same tolerance.